crates/levm/src/call_frame.rs

```diff
@@ -32,4 +32,24 @@ impl CallFrame {
     pub fn pc(&self) -> usize {
         self.pc
     }
+
+    pub fn jump(&mut self, jump_address: U256) {
+        if !self.valid_jump(jump_address) {
+            // probably should halt/panic
+            dbg!("Invalid jump");
+            return;
+        }
+        self.pc = jump_address.as_usize() + 1;
+    }
+
+    fn valid_jump(&self, offset: U256) -> bool {
+        // In the future this should be the Opcode::Invalid and halt
+        let opcode = self
+            .bytecode
+            .get(offset.as_usize())
+            .copied()
+            .map(Opcode::from)
+            .unwrap_or(Opcode::STOP);
+        opcode == Opcode::JUMPDEST
+    }
```

Comment:
Here you're using `jump_address` and `offset` as names for the same thing. Let's choose one of them to represent the same thing or find another name that suits both functions.

Reply:
Oops missed that one, should be `jump_address` for both (in evm.codes they call it "offset" when in reality it doesn't work like one)